app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -173,15 +196,9 @@ def handle_failed_status(enrollment, response)
     enrollment_outcomes[:enrollments_failed] += 1
     analytics(user: enrollment.user).idv_in_person_usps_proofing_results_job_enrollment_updated(
       **enrollment_analytics_attributes(enrollment, complete: true),
-      failure_reason: response['failureReason'],
```

Comment:
just for my understanding - do we not need this anymore because we're passing err.response response_analytics_attributes?

Reply:
oh yes i looked at response_analytics_attributes again

Reply:
Yep, you got it.  `failure_reason` [is included in](https://github.com/18F/identity-idp/pull/6967/files#diff-3999219037053b6ad66255d8f7e863be02a97e1543c499ad75f3bd98b28a1ed7R36) `response_analytics_attributes`.